src/firefly/java/edu/caltech/ipac/firefly/server/query/EmbeddedDbProcessor.java

```diff
@@ -162,7 +159,8 @@ public DataGroupPart getData(ServerRequest request) throws DataAccessException {
             } catch (Exception e) {
                 // table data exists.. but, bad grammar when querying for the resultset.
                 // should return table meta info + error message
-                DataGroup dg = EmbeddedDbUtil.execQuery(DbAdapter.getAdapter(treq), dbFile, "select * from data limit 0", "data");
+                // limit 0 does not work with oracle-like syntax
+                DataGroup dg = EmbeddedDbUtil.execQuery(DbAdapter.getAdapter(treq), dbFile, "select * from data where ROWNUM < 1", "data");
```

Comment:
Regarding LIMIT 0 at the end of SELECT statement. 

After reading https://sourceforge.net/p/hsqldb/bugs/1100/ I thought that hsqldb is expected to return 0 rows with LIMIT 0. But it does not. May be LIMIT 0 is not supported because we are using oracle-like syntax (`sql.syntax_ora=true`) and oracle does not support LIMIT? Anyway, I thought we better have this note, in case we decide to use limit again in the future. The behavior is unexpected.